packages/client/src/pages/ForumTopic/elements/EmojiList/EmojiList.tsx

```diff
@@ -0,0 +1,83 @@
+import { useAppDispatch, useAppSelector } from '@store/hooks'
+import { userSelectors } from '@store/user'
+import styles from './emojiList.module.css'
+import { forumActions, forumSelectors } from '@store/forum'
+import { EmojiProps } from './types'
+import { FC } from 'react'
+import { ReactionDTO } from '@api/reaction/types'
+import { addReaction } from '@store/topic/thunk'
+
+const EmojiList: FC<EmojiProps> = ({ topicId }) => {
+    const userData = useAppSelector(userSelectors.selectData)
+    const emojis = useAppSelector(forumSelectors.selectEmojis)
+    const topicEmoji = useAppSelector(forumSelectors.selectTopicEmoji)
+    const dispatch = useAppDispatch()
+    const handleEmojiClick = (
+        event: React.MouseEvent<HTMLElement>,
```

Comment:
Здесь реакт ниоткуда не импортируется, нужно либо добавить его импорт либо так
```suggestion
        event: MouseEvent<HTMLElement>,
```